modules/apps/segments/segments-web/webpack.dev.config.js

```diff
@@ -0,0 +1,65 @@
+const path = require('path')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+
+
+module.exports = {
+	entry: {
+		'ODataParser': path.resolve(__dirname,'src','main','resources','META-INF','resources','js','libs','ODataParser.es.js'),
+		'index.dev': path.resolve(__dirname,'src','main','resources','META-INF','resources','js','index.dev.js'),
+		'style': path.resolve(__dirname,'src','main','resources','META-INF','resources','css','main.scss'),
```

Comment:
Rather than making all of these css entry points, you might try just importing the css files into the `index.dev.js` file. The less config inside of webpack will make it easier for future devs reading through this.